src/main/java/edu/wisc/my/rssToJson/filter/XmlFilter.java

```diff
@@ -0,0 +1,35 @@
+package edu.wisc.my.rssToJson.filter;
+ import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+ public class XmlFilter {
+     protected final Logger logger = LoggerFactory.getLogger(getClass());
+     public static iFilter getXmlFilter(String filterName){
+        try{ 
+          String filterClass = XmlFilter.toTitleCase(filterName) + "Filter";
+          String pkg = new CurrentClassGetter().getPackageName();
+          iFilter filter = (iFilter) Class.forName(pkg + "." +filterClass).newInstance();
+          return filter;
+       } catch (Exception e){
+           return null;
+       }
+    }
+    public static class CurrentClassGetter extends SecurityManager {
+    public String getPackageName() {
+      return getClassContext()[1].getPackage().getName(); 
+    }
+  }
+     private static String toTitleCase(String filterNameIn){
```

Comment:
Neat, but it only handles a narrow case of one word keys? (like I don't think it would handle `working-at-uw`)

Maybe want to pull in commons-text for this? https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/CaseUtils.html#toCamelCase(java.lang.String,boolean,char...)